src/foam/nanos/crunch/cronjobs

```diff
@@ -16,7 +16,7 @@ p({
     cron.execute(x);
     print("done");
   """,
-  "enabled":true
+  "enabled":false
```

Comment:
Disable during medusa testing as we have a midnight cron bug - run repeated in the midnight hour.